pkg/resource/apply.go

```diff
@@ -30,6 +37,8 @@ var (
 	UnknownApplyResult ApplyResult = "unknown"
 )
 
+const fieldTooLong metav1.CauseType = "FieldValueTooLong"
```

Comment:
I did not understand why only this const is declared outside const() block above